source/includes/api/_dataset.md

```diff
@@ -74,81 +76,140 @@ curl -X GET https://api.resourcewatch.org/v1/dataset
+Here's the comprehensive list of filters supported by datasets:
+ 
 
-The API currently supports sorting by means of the `sort` parameter. Sorting can be done using any field from the dataset, as well as `user.name` and `user.role` (sorting by user data is restricted to ADMIN users).
+Filter         | Description                                                                  | Type        | Expected values
+-------------- | ---------------------------------------------------------------------------- | ----------- | -------------------------------------------------------------------------------
+name           | Filter returned datasets by the name of the dataset.                         | String      | any valid text
+slug           | Filter returned datasets by the slug of the dataset.                         | String      | any valid text
+type           | Filter returned datasets by dataset type.                                    | String      | any valid text
+subtitle       | Filter returned datasets by dataset subtitle.                                | String      | any valid text
+application    | Applications associated to this dataset.                                     | Array       | any valid text
+applicationConfig |                                                                           | Object      | `true` or `false`
+dataPath       |                                                                              | String      | any valid text
+attributesPath |                                                                              | String      | any valid text
+connectorType  | Filter returned datasets by the type of connector used.                      | String      | `rest`, `document` or `wms`
+provider       | Dataset provider this include inner connectors and 3rd party ones            | String      | [Check the available providers when creating a dataset](index-rw.html#creating-a-dataset)
+connectorUrl   |                                                                              | String      | any valid text
+sources        |                                                                              | Array       | any valid text
+tableName      |                                                                              | String      | any valid text
+userId         | Filter results by the owner of the dataset. Does not support regexes.        | String      | valid user id
+status         | Filter results by the current status of the dataset.                         | String      | `pending`, `saved` or `failed`
+overwrite      | If the data can be overwritten (only for being able to make dataset updates) | Boolean     | `true`or `false`
+verified       | If this dataset contains data that is verified using blockchain              | Boolean     | `true`or `false`
+errorMessage   | If this dataset contains data that is verified using blockchain              | String      | any valid text
```

Comment:
Description is wrong for `errorMessage` and `mainDateField`.

Reply:
fixed